src/constants/selectOptions.ts

```diff
@@ -0,0 +1,24 @@
+export const CATEGORY_WITH_ENTIRE: CategoryWithEntire[] = [
+  "전체",
+  "한식",
+  "중식",
+  "일식",
+  "아시안",
+  "양식",
+  "기타",
+];
+
+export const SORT_STANDARD: SortStandard[] = ["이름순", "거리순"];
+
+export const CATECORY: Category[] = [
```

Comment:
소소하게 여기 오타가 있네요 ㅎㅎ Code Spell Checker라는 VSC 익스텐션이 있는데 추천 드립니다
```suggestion
export const CATEGORY: Category[] = [
```